src/YamlValidator/Constants.cs

```diff
@@ -11,4 +11,8 @@ public static class Constants
     public const string JsonFileExtension = ".json";
 
     public const string Verbose = "verbose";
+
+    // runtime constants
+    // default schema path
+    public static readonly string[] DefaultSchemaPath = [".", "schema", "pa.yaml-schema.json"];
```

Comment:
You can also just do the `Path.Combine(...)` here directly:
```suggestion
    public static readonly string DefaultSchemaPath = Path.Combine(".", "schema", "pa.yaml-schema.json");
```